.ci_cd/ci_cd.json

```diff
@@ -13,7 +13,8 @@
 	"release": {
 		"distribute": {
 			"docker": "latest,$version",
-      "maven": "$version"
+			"maven": "$version",
+			"npm": "$version"
 		},
 		"deploy": {
```

Comment:
There is no 'production' environment in this repo. We might want to always deploy the latest release to the 'staging' and 'demo' environments instead?